### DISCUSSION TITLE
【English colearning】Peer Accountability

### DISCUSSION THREAD

**Discussion by Demian101** [creatorsand/.github#50] [Ideas]
##  **伙伴责任制**

适合 2 个研究领域相同的人，结伴 colearning

其中一人选定研究材料，发 Github issue 寻求队友，两人约定本周学习内容，约定本周视频一起 share & 口语分享。

举例：
- Topic: 本周学习 [An overview of the Groth 16 proof system](https://blog.lambdaclass.com/)
- Prove 形式: 录制视频（证明自己确实看了）

分享完之后，使用 https://github.com/orgs/CreatorsDAO/discussions/48 工具 improve 

----

Show me your bookshelf and give me a book.

Mike: make a promise to learn [think in english course]

- Punishment:  Show me your bookshelf and give me a book
- if both achieved: do it !

**Comment by v1xingyue** [comment]
This is  awesome!!
